01/app.js

```diff
@@ -1,2 +1,10 @@
 const randomNumber = Math.round(Math.random() * 20);
-console.log(randomNumber);
+console.log('Wylosowana liczba, to: ' + randomNumber);
+
+if (randomNumber > 5) {
+	for (let i = 5; i <= randomNumber; i++) {
+		console.log(i);
+	}
```

Comment:
Trochę duże te odstępy od lewej krawędzi - standardowo to 2 lub 4 znaki spacji :)